modules/apps/frontend-js/frontend-js-web/package.json

```diff
@@ -39,7 +39,8 @@
 		"uuid": "^3.3.2"
 	},
 	"devDependencies": {
-		"babel-loader": "^8.0.5"
+		"babel-loader": "^8.0.5",
+		"jest-xml-matcher": "^1.1.1"
```

Comment:
For some reason, my test cannot find new library:
```
  ● Test suite failed to run

    Cannot find module 'jest-xml-matcher' from 'format_xml.es.js'

      2 |
      3 | import formatXML from '../../../../src/main/resources/META-INF/resources/liferay/
util/xmlformatter/format_xml.es';
    > 4 | import 'jest-xml-matcher';
        | ^

```
I made a `gradle clean deploy` of frontend-js-web.